src/generate.jl

```diff
@@ -0,0 +1,74 @@
+function new(pkg::String; minimal::Bool=true)
+    minimal == true || cmderror("only minimal package generation implemented")
```

Comment:
What do you imagine the difference between minimal and maximal generation to be?

Reply:
`.travis`, `appveyor`, `.codecov`, `test/runtests.jl`, `docs/make.jl`, git initialization etc...

Reply:
I removed this kwarg for now.